skrub/_repr.py

```diff
@@ -0,0 +1,103 @@
+import itertools
```

Comment:
could we name this module something more explicit like "_sklearn_html_repr_utils" or similar?